### PR TITLE
Add backup keyserver option for the install script

### PR DESCRIPTION
**Pull request by remicalixte**
### Motivation

We should make sure our APT repo's public GPG key is available from another keyserver (pool) than just keyserver.ubuntu.com so that, when the latter is down.

### CODE REVIEW

**Review by remeh — DISMISSED** [review]
👍

File: cmd/agent/install_script.sh [hunk 1]
```diff
@@ -133,6 +133,12 @@ if [ $OS = "RedHat" ]; then
     $sudo_cmd yum -y clean metadata
     $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install datadog-agent || $sudo_cmd yum -y install datadog-agent
 elif [ $OS = "Debian" ]; then
+
+    keyserver=hkp://keyserver.ubuntu.com:80
```
Comment (olivielpeau) [inline cmd/agent/install_script.sh]: nit: define variable with quotes: `keyserver="hkp://keyserver.ubuntu.com:80"`
(same thing below)

File: cmd/agent/install_script.sh [hunk 2]
```diff
@@ -133,6 +133,12 @@ if [ $OS = "RedHat" ]; then
...
+    if [ -n "$USE_APT_BACKUP_KEYSERVER" ]; then
+    keyserver=hkp://pool.sks-keyservers.net:80
```
Comment (olivielpeau) [inline cmd/agent/install_script.sh]: nit: inconsistent indentation

File: cmd/agent/install_script.sh [hunk 3]
```diff
@@ -133,6 +133,12 @@ if [ $OS = "RedHat" ]; then
     $sudo_cmd yum -y clean metadata
     $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install datadog-agent || $sudo_cmd yum -y install datadog-agent
 elif [ $OS = "Debian" ]; then
+
+    keyserver=hkp://keyserver.ubuntu.com:80
+    if [ -n "$USE_APT_BACKUP_KEYSERVER" ]; then
```
Comment (olivielpeau) [inline cmd/agent/install_script.sh]: we should avoid using env vars directly in the middle of the script, it makes it very hard to understand the vars that the script supports. Let's define a local variable like `use_apt_backup_keyserver` at the beginning of the script and then use it here.

File: cmd/agent/install_script.sh [hunk 4]
```diff
@@ -144,7 +150,7 @@ elif [ $OS = "Debian" ]; then
     fi
     printf "\033[34m\n* Installing APT package sources for Datadog\n\033[0m\n"
     $sudo_cmd sh -c "echo 'deb https://apt.${repo_url}/ stable 6' > /etc/apt/sources.list.d/datadog.list"
-    $sudo_cmd apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 382E94DE
+    $sudo_cmd apt-key adv --recv-keys --keyserver $keyserver 382E94DE
```
Comment (olivielpeau) [inline cmd/agent/install_script.sh]: prefer using `"${keyserver}"` for var interpolation

**Review by olivielpeau — COMMENTED** [review]
some style nits (should be caught by some kind of linter really 😄 ), other than that LGTM!

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**